Export ROIs directly to ImageJ

mrariden opened the pull request:
Allows exporting outlines to .zip archive of .roi files using roifile ImageJ as mention in #581. This introduces a new dependency, [roifile](https://github.com/cgohlke/roifile). 

- Left the "Save Outlines" functionality as it was both in the GUI and in the command line args
- Added `--save_rois` command line flag
- Added GUI button similar to existing "save outlines" functionality

Did not update docs (how/should I do this?)

carsen-stringer commented:
Thanks you so much, this looks great and super useful! I just need to test it, will get to it soon. The documentation can be added here: https://github.com/MouseLand/cellpose/blob/main/docs/outputs.rst

mrariden commented:
Cellpose segmentation:
![image](https://github.com/MouseLand/cellpose/assets/54959329/67d09f2a-f700-4ad5-80fd-b189c3069158)

Save the ROIs:
Going into CP GUI -> save outlines as .zip 

Use the ROIs in FIJI/ImageJ:
1) Open image
2) Import ROIs using File -> open. Recent versions will autodetect that the file is an roi archive
3) Click 'show all' to display the rois on the image. 
![image](https://github.com/MouseLand/cellpose/assets/54959329/70ed9124-ff2c-4d3f-abc2-cf7f4c4ea9f1)

Note that the `roifile` library appends rois to existing archives. So, if the user removes rois after saving the removed rois will still be there when the user saves again. I circumvented this by deleting the file before writing, but this may not be ideal for images with many ROIs.

I added ctrl+R shortcut for the function in the GUI, and updated relevant docs.

carsen-stringer commented:
thanks this looks great!
